otter/models/cass.py

```diff
@@ -694,7 +694,7 @@ def with_timestamp(self, func):
         """
         @functools.wraps(func)
         def wrapper(*args):
-            d = get_client_ts(self.reactor)
+            d = defer.maybeDeferred(get_client_ts, self.reactor)
```

Comment:
Non-blocking - since it's a synchronous function, can this just call `func(get_client_ts(self.reactor), *args)` and return the result?  It looks like all the functions that are decorated returned Deferreds anyway, and the docstring does not say that this will always wrap the function result in a Deferred.